docs/core/idioms/pause-for-approval.md

```diff
@@ -0,0 +1,53 @@
+# Pause for Approval
+
+A manual-only trigger that has been applied to a task will pause the task once it is reached. The task can only be resumed by an individual or event, either from the Prefect UI or GraphQL API. 
```

Comment:
I suggest adding a little more context prior to this line; something like:

There are many situations where a task requires some form of manual approval in order to run successfully.  Prefect has two built-in mechanisms for achieving this pattern: 
- a manual-only trigger that can be configured on a per task basis, and prevents that task from running unless a user approves it
- a `PAUSE` signal that can be raised programmatically when certain conditions are met

Both of these mechanisms mean that the task can only be resumed by an individual or event, either from the Prefect UI or GraphQL API.

Reply:
See changes.